resources/views/pages/show.blade.php

```diff
@@ -1,5 +1,10 @@
 @extends('tri-layout')
 
+@push('social-meta')
+    <meta property="og:description" content="{{ Str::words($page->html, 50, '...') }}">
```

Comment:
Instead of `html`, what do you think of using the `text` property instead? That would avoid tog:description` having HTML content.

Reply:
Agreed, updated to use `text` property.